### PR TITLE
Risk of Rain 2 dynamic item pool

### PR DESCRIPTION
**Pull request by alwaysintreble**
Added dynamic item weights pool, presets for the new pool, and a toggle option to prevent any possible confusion. Also added descriptions to the alttp options

### CODE REVIEW

**Review by Ijwu — APPROVED** [review]

**Comment by Berserker66** [comment]
The way that "chaos weights" are implemented they'd be the same always during runtime of the program, in case of the webhost that would mean they remain the same for weeks at a time.

**Comment by Ijwu** [comment]
Fixed merge conflict. LGTM, awaiting second review before merging.

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
